modules/data-entry/src/main/frontend/src/questionnaire/Subject.jsx

```diff
@@ -218,7 +229,16 @@ function SubjectContainer(props) {
 
   return (
     subject && <React.Fragment>
-      <SubjectMember classes={classes} id={id} level={currentLevel} data={subject} maxDisplayed={maxDisplayed} pageSize={pageSize} onDelete={() => {setDeleted(true)}} childSubjects={relatedSubjects}/>
+      <SubjectMember
+        classes={classes}
+        id={id}
+        level={currentLevel}
+        data={subject}
+        maxDisplayed={maxDisplayed}
+        pageSize={pageSize}
+        onDelete={() => {setDeleted(true)}}
+        childSubjects={relatedSubjects}
+        fetchSubjectData={fetchSubjectData}/>
```

Comment:
Very minor code style suggestion:
```suggestion
        fetchSubjectData={fetchSubjectData}
        />
```